### PR TITLE
Google analytics

### PR DESCRIPTION
**Pull request by sri2k1us**
image

### CODE REVIEW

File: src/gtag.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const GA_TRACKING_ID = "G-YXTRXCKR7T";
```
Comment (ash3rz) [inline src/gtag.js:1]: Should this also be in the config?

File: src/pages/_app.js [hunk 2]
```diff
@@ -90,6 +92,16 @@ function MyApp({ Component, pageProps }) {
         queries: { refetchOnWindowFocus: false, retry: false },
     };
 
+    useEffect(() => {
+        const handleRouteChange = (url) => {
+            gtag.pageview(url);
+        };
+        router.events.on("routeChangeComplete", handleRouteChange);
+        return () => {
+            router.events.off("routeChangeComplete", handleRouteChange);
```
Comment (ash3rz) [inline src/pages/_app.js:101]: 👍 I always forget about the return function with useEffect

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Review by ianmcorvidae — APPROVED** [review]
Looks good as far as I can tell! I'm not totally sure about what GA provides us as far as looking at this data, but it seems like we're following best practices anyway.

**Comment by sri2k1us** [comment]
Thanks for reviews.

**Pull request merged**